src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,37 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
pls destructure movie

Reply:
I don't no why, but when I destructure movie in the Live Service all looks good and also work but in this case npm test maybe can't see element like title, description etc. and all the test failed

Reply:
please ask this question in fe_chat

Reply:
@ivan-baranovskyi try destructuring `movie` inside the component, not directly in props